src/main/java/net/fabricmc/meta/data/VersionDatabase.java

```diff
@@ -43,6 +45,10 @@ public class VersionDatabase {
 	private List<MavenBuildVersion> loader;
 	public List<MavenUrlVersion> installer;
 
+	private static final ArrayList<String> incorrectVersions = new ArrayList<>();
+
+	private static final Logger LOGGER = new SimpleLoggerFactory().getLogger(VersionDatabase.class.getName());
```

Comment:
NIT: move these static fields above the none static ones.

Reply:
I should really set up intellij to rearrange my class members for me